packages/c-code/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@chakra-ui/c-code",
+  "description": "Chakra UI Vue | Code is a component used to display inline code. It is composed from the Box component with a font family of mono for displaying code",
+  "version": "0.0.1-alpha.0",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/types/index.d.ts",
+  "typings": "dist/types/index.d.ts",
+  "author": "Vivek Wadhwani <devivekw@pm.me>",
```

Comment:
I suggest you leave this field as it was, and add another field in the `package.json` called `"contributors": ["Vivek Wadhwani <devivekw@pm.me>"],` and add it there. 🙏🏽

Reply:
We will develop a better system to document all Chakra UI contributors repository better on the repository and on the website using `all-contributors API`. I haven't yet gotten around to doing it but will do so next week.

Thanks a lot for the work done here. ❤️

Reply:
I appreciate it! I look forward to working on this package more :)